### PR TITLE
[ADD] estate: add spanish translations on estate model T#70778

### PR DESCRIPTION
**Pull request by Gilmaciasr**
Now it's available to activate español mx translations for the estate module.
Link to the lesson:
https://www.odoo.com/documentation/16.0/developer/howtos/translations.html
Images of the result:
image
image
image
image
image
image
image
image

### CODE REVIEW

File: estate/i18n/es_MX.po [hunk 1]
```diff
@@ -0,0 +1,687 @@
...
+msgid "'Property offers report - %s' % (object.name or 'Property')"
+msgstr ""
+"'Reporte de oferta de propiedades - %s' % (object.name or 'Propiedad')"
+
+#. module: estate
+#: model:ir.actions.report,print_report_name:estate.estate_property_action_offers_per_salesman_report
+msgid "'Salesman properties offers report - %s' % (object.name or 'Salesman')"
+msgstr ""
+"'Reporte de ofertas de propiedades por vendedor- %s' % (object.name or "
+"'Vendedor')"
```
Comment (JesusValdez96) [inline estate/i18n/es_MX.po:31]:
```suggestion
"'Reporte de ofertas de propiedades por vendedor - %s' % (object.name or "
"'Vendedor')"
```

File: estate/i18n/es_MX.po [hunk 2]
```diff
@@ -0,0 +1,687 @@
...
+
+#. module: estate
+#: model:ir.ui.menu,name:estate.estate_menu_advertisements
+msgid "Advertisements"
+msgstr "Anuncios "
+
+#. module: estate
+#: model:res.groups,name:estate.estate_group_user
+msgid "Agent"
+msgstr "Agente de ventas"
```
Comment (JesusValdez96) [inline estate/i18n/es_MX.po:83]:
```suggestion
msgstr "Agente"
```

File: estate/i18n/es_MX.po [hunk 3]
```diff
@@ -0,0 +1,687 @@
...
+
+#. module: estate
+#: model:ir.model,name:estate.model_estate_property_offer
+msgid ""
+"Is an ammount a potential buyer offers to the seller.                     "
+"The offer can be lower or higher than the expected price."
+msgstr ""
+"Es una cantidad que un posible compratdor puede ofertas al "
+"vendedor .                     La oferta puede ser menor o mayor que el "
+"precio previsto."
```
Comment (JesusValdez96) [inline estate/i18n/es_MX.po:266]:
```suggestion
msgid ""
"Is an amount a potential buyer offers to the seller.                     "
"The offer can be lower or higher than the expected price."
msgstr ""
"Es una cantidad que un posible comprador puede ofertar al vendedor."
"                     La oferta puede ser menor o mayor que el precio "
"previsto."
```

File: estate/i18n/es_MX.po [hunk 4]
```diff
@@ -0,0 +1,687 @@
...
+
+#. module: estate
+#: model_terms:ir.ui.view,arch_db:estate.res_users_view_form
+msgid "Real estate properties"
+msgstr "Propiedades de bienes raíces"
+
+#. module: estate
+#: model:ir.model,name:estate.model_estate_property
+msgid "Real state property"
+msgstr "Propiedade de bienes raíces"
```
Comment (JesusValdez96) [inline estate/i18n/es_MX.po:485]:
```suggestion
msgstr "Propiedades de bienes raíces"
```

File: estate/i18n/es_MX.po [hunk 5]
```diff
@@ -0,0 +1,687 @@
...
+#: model_terms:ir.ui.view,arch_db:estate.estate_property_view_form
+msgid "Sold"
+msgstr "Vendida"
+
+#. module: estate
+#. odoo-python
+#: code:addons/estate/models/estate_property.py:0
+#, python-format
+msgid "Sold properties cannot be canceled"
+msgstr "Las propiedades vendidas no puedes ser canceladas"
```
Comment (JesusValdez96) [inline estate/i18n/es_MX.po:538]:
```suggestion
msgstr "Las propiedades vendidas no pueden ser canceladas"
```

File: estate/i18n/es_MX.po [hunk 6]
```diff
@@ -0,0 +1,687 @@
...
+
+#. module: estate
+#. odoo-python
+#: code:addons/estate/models/estate_property.py:0
+#, python-format
+msgid "Sold properties cannot receive offers."
+msgstr "Las propiedades vendidas no puede recibir ofertas"
```
Comment (JesusValdez96) [inline estate/i18n/es_MX.po:545]:
```suggestion
msgstr "Las propiedades vendidas no pueden recibir ofertas"
```

File: estate/i18n/es_MX.po [hunk 7]
```diff
@@ -0,0 +1,687 @@
...
+#. module: estate
+#: model:ir.model.fields,field_description:estate.field_estate_property__tag_ids
+#: model_terms:ir.ui.view,arch_db:estate.estate_property_tag_view_tree
+msgid "Tag"
+msgstr "Etiqueta"
+
+#. module: estate
+#: model:ir.model.constraint,message:estate.constraint_estate_property_tag_unique_tag
+msgid "Tag name should be unique."
+msgstr "El nombre de la etiqueta debe ser unico"
```
Comment (JesusValdez96) [inline estate/i18n/es_MX.po:573]:
```suggestion
msgstr "El nombre de la etiqueta debe ser único"
```

File: estate/i18n/es_MX.po [hunk 8]
```diff
@@ -0,0 +1,687 @@
...
+
+#. module: estate
+#: model:ir.model.fields,field_description:estate.field_estate_property_offer__validity
+msgid "Validity"
+msgstr "Validez"
+
+#. module: estate
+#: model_terms:ir.ui.view,arch_db:estate.estate_property_offer_view_tree
+msgid "Validity (Days)"
+msgstr "Validez (Dias)"
```
Comment (JesusValdez96) [inline estate/i18n/es_MX.po:660]:
```suggestion
msgstr "Validez (Días)"
```

File: estate/i18n/es_MX.po [hunk 9]
```diff
@@ -0,0 +1,687 @@
...
+
+#. module: estate
+#: model_terms:ir.ui.view,arch_db:estate.estate_property_offer_view_form
+#: model_terms:ir.ui.view,arch_db:estate.estate_property_view_form
+#: model_terms:ir.ui.view,arch_db:estate.offers_table_by_property_report
+msgid "Validity (days)"
+msgstr "Validez (dias)"
```
Comment (JesusValdez96) [inline estate/i18n/es_MX.po:667]:
```suggestion
msgstr "Validez (días)"
```

File: estate/i18n/estate.pot [hunk 10]
```diff
@@ -0,0 +1,675 @@
+# Translation of Odoo Server.
```
Comment (JesusValdez96) [inline estate/i18n/estate.pot:1]: this file is not neccesary

**Review by JesusValdez96 — CHANGES_REQUESTED** [review]

**Comment by Gilmaciasr** [comment]
@JesusValdez96 Done!

**Pull request merged**